commit 75fb273bead026ec45824c7f32d439985d9fdd49
Author: Paul Johnston <pcj@stack.build>
Date:   Sun Oct 5 15:10:16 2025 -0600

    require gbitint

diff --git a/javascript/soyutils_usegoog.js b/javascript/soyutils_usegoog.js
index 29ec261a5..bf1794f33 100644
--- a/javascript/soyutils_usegoog.js
+++ b/javascript/soyutils_usegoog.js
@@ -39,6 +39,7 @@ const {Message} = goog.requireType('jspb');
 const {SafeHtml, SafeScript, SafeStyleSheet, TrustedResourceUrl, isUrl, unwrapHtml, unwrapResourceUrl, unwrapScript, unwrapStyleSheet, unwrapUrl} = goog.require('google3.third_party.javascript.safevalues.index');
 const {SanitizedContent, SanitizedContentKind, SanitizedCss, SanitizedHtml, SanitizedHtmlAttribute, SanitizedJs, SanitizedTrustedResourceUri, SanitizedUri} = goog.require('goog.soy.data');
 const {compareBigInt} = goog.require('google3.javascript.common.bigint.index');
+const {gbigint} = goog.requireType('soy.types.gbigint');
 const {htmlSafeByReview} = goog.require('google3.third_party.javascript.safevalues.restricted.reviewed');
 const {isReadonly} = goog.require('google3.javascript.apps.jspb.types.is_readonly');
 
