commit 916f1d26671e8d30f3ff437eafa2760f7ebb6385
Author: Paul Johnston <pcj@stack.build>
Date:   Sun Oct 5 15:06:56 2025 -0600

    remove proto immutable

diff --git a/javascript/soyutils_usegoog.js b/javascript/soyutils_usegoog.js
index 6a63f0c07..29ec261a5 100644
--- a/javascript/soyutils_usegoog.js
+++ b/javascript/soyutils_usegoog.js
@@ -39,7 +39,6 @@ const {Message} = goog.requireType('jspb');
 const {SafeHtml, SafeScript, SafeStyleSheet, TrustedResourceUrl, isUrl, unwrapHtml, unwrapResourceUrl, unwrapScript, unwrapStyleSheet, unwrapUrl} = goog.require('google3.third_party.javascript.safevalues.index');
 const {SanitizedContent, SanitizedContentKind, SanitizedCss, SanitizedHtml, SanitizedHtmlAttribute, SanitizedJs, SanitizedTrustedResourceUri, SanitizedUri} = goog.require('goog.soy.data');
 const {compareBigInt} = goog.require('google3.javascript.common.bigint.index');
-const {defaultImmutableInstance} = goog.require('jspb.immutable_message');
 const {htmlSafeByReview} = goog.require('google3.third_party.javascript.safevalues.restricted.reviewed');
 const {isReadonly} = goog.require('google3.javascript.apps.jspb.types.is_readonly');
 
@@ -2612,9 +2611,6 @@ const /** boolean */ SOY_CREATED_PROTOS_ARE_IMMUTABLE = false;
  * @return {?}
  */
 function $$emptyProto(/** function(new:Message,?Array<?>=)*/ ctor) {
-  if (SOY_CREATED_PROTOS_ARE_IMMUTABLE) {
-    return defaultImmutableInstance(ctor);
-  }
   return new ctor();
 }
 
@@ -2624,9 +2620,6 @@ function $$emptyProto(/** function(new:Message,?Array<?>=)*/ ctor) {
  * @return {?}
  */
 function $$maybeMakeImmutableProto(/** !Message*/ message) {
-  if (SOY_CREATED_PROTOS_ARE_IMMUTABLE) {
-    return message.toImmutable();
-  }
   return message;
 }
 
