--- a/soyutils_usegoog.js
+++ b/soyutils_usegoog.js
@@ -149,7 +149,7 @@ soydata.SanitizedHtml.from = function(value) {
   }
   if (value instanceof goog.html.SafeHtml) {
     return soydata.VERY_UNSAFE.ordainSanitizedHtml(
-        goog.html.SafeHtml.unwrap(value), value.getDirection());
+        goog.html.SafeHtml.unwrap(value));
   }
   return soydata.VERY_UNSAFE.ordainSanitizedHtml(
       soy.esc.$$escapeHtmlHelper(String(value)), soydata.getContentDir(value));

